src/main/java/com/mraof/minestuck/block/HorseClockBlock.java

```diff
@@ -0,0 +1,141 @@
+package com.mraof.minestuck.block;
+
+import com.mraof.minestuck.block.machine.MachineMultiblock;
+import com.mraof.minestuck.block.machine.MultiMachineBlock;
+import com.mraof.minestuck.blockentity.HorseClockBlockEntity;
+import com.mraof.minestuck.blockentity.MSBlockEntityTypes;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.EntityBlock;
+import net.minecraft.world.level.block.RenderShape;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.entity.BlockEntityTicker;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.BlockStateProperties;
+import net.minecraft.world.level.block.state.properties.IntegerProperty;
+import net.minecraft.world.phys.BlockHitResult;
+
+import javax.annotation.Nullable;
+import java.util.Random;
+
+public class HorseClockBlock extends MultiMachineBlock implements EntityBlock
+{
+	public static final IntegerProperty POWER = BlockStateProperties.POWER;
```

Comment:
Are all three blocks making use of the power property? Also, when writing a javadoc, you could add a description on how the power property is used.

Reply:
For the javadoc on this class, you could just refer to the block entity class if you don't want to add anything to the block class specifically.